### ISSUE TITLE
issue with static mapping

### ISSUE THREAD

**Issue by juanbrujo** [shootaroo/jit-grunt#35]
Still can`t get this https://github.com/shootaroo/jit-grunt#static-mappings and it is precisely with grunt-spritesmith task. My sprite task:

```
sprite:{
    all: {
        src: 'src/images/sprites/*.png',
        destImg: 'src/images/sprites.png',
        destCSS: 'src/less/inc/sprites.less',
        algorithm: 'binary-tree',
        padding: 2
    }
 },
```

**Comment by henrahmagix** [comment]
What is your static mappings config? This should work for you:
```js
{
    sprite: 'grunt-spritesmith'
}
```

**Comment by juanbrujo** [comment]
@henrahmagix default config, I'll start debug now if there's conflict with another task.

**Comment by juanbrujo** [comment]
fixed, it was grunt-spritesmith < 4.0.0.
